src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
```

Comment:
Generally according to the mates checklist you should make empty lines between siblings (here `a` and `nav`), below example how it should be interpreted:
```html
<div>
  <div></div>
  
  <section>
    <p></p>
  </section>

  <p></p>
</div>

<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```
As you can see there is an exception to that rule, when you have repeated elements like `li`'s in `ul` there is no need for the empty lines.